packages/mobile/src/screens/home/components/product-section/product-section.tsx

```diff
@@ -10,6 +10,7 @@ import { globalStyles } from '~/styles/styles';
 import { ProductDto } from '@vse-bude/shared';
 import { StyleProp, ViewStyle } from 'react-native';
 import { useCustomTheme } from '~/hooks/hooks';
+import { styles } from '../../styles';
```

Comment:
It is better to create separate styles file for product-section, which will contain styles only for product section

Reply:
fixed